static/src/javascripts-legacy/bootstraps/enhanced/main.js

```diff
@@ -163,10 +163,17 @@ define([
         }
 
         // use a #force-sw hash fragment to force service worker registration for local dev
-        if ((window.location.protocol === 'https:' && config.page.section !== 'identity') || window.location.hash === '#force-sw') {
+        if ((window.location.protocol === 'https:' && config.page.section !== 'identity') || window.location.hash.indexOf('force-sw') > -1) {
```

Comment:
Does this suffer from a race condition as requests for ias will begin in commercial but this isn't registered until enhanced? I suppose that's ok as most ads are lazy loaded so we might miss sticky top but catch the others?